docker/start.py

```diff
@@ -95,6 +96,9 @@ def start_kms():
 
 # Main
 if (__name__ == "__main__"):
+  # Do not import for minimal
+  if os.getenv('TYPE', '') != "MINIMAL":
+    import sqlite_web
```

Comment:
Why do we even need to import anything? As far as I see the command is executed on the console regardless.

Reply:
keeping two dockerfiles but an unified start.py, this is to not import a lib that was not installed.
the sqlite_web package was cloned from github, now is installed from pip.

From console, or "full" image, the sqlite_web is downloaded and installed (requirements.txt)

Reply:
But... `py-kms` nor the entrypoint are using this module directly. They all only use the console command! So, please remove this import (or enlighten me, why we would need it).

Reply:
will do. no module use as you said.